python/main.py

```diff
@@ -346,10 +352,36 @@ def __convert_recipients_to_list(recipients_str):
     return recipients_str.translate(remove_chars_trans).split(',')
 
 
-def invoke_lambda(error_msg):
+def invoke_lambda(args):
 
     lambda_client = boto3.client('lambda', LAMBDA_REGION)
-    lambda_client.invoke(FunctionName=LAMBDA_NAME, Payload={"message": error_msg})
+    lambda_client.invoke(FunctionName=LAMBDA_NAME, Payload=json.dumps(args))
+
+
+def __email_validation():
```

Comment:
I think its better to do the validation on startup and not just when you find your first error